packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -22,7 +22,8 @@
       background: var(--terra-consumer--mobile-nav-background, #383f42);
       box-shadow: 0 1px 3px;
       display: block;
-      opacity: 0.95;
+      opacity: 0.8;
+      width: 100vw;
```

Comment:
Reviewed the Nav with Scott on Friday and he wanted the opacity changed to 0.8. I will remove this Nav UI changes.